readme.md

```diff
@@ -75,6 +75,37 @@ The codebase also has a number of reference examples.  These are all marked with
 
 `MAINTENANCE_MODE`: Set this flag on a server environment to stop all user requests to the site, such as when you need to make substantial server updates or run a complex database migration.
 
+
+## Deployment
+Prerequisites:
+- AWS CLI
+```
+pip install awscli
```

Comment:
At this point, we should make an elastic_beanstalk feature flag, put that on the eb-config file, and then put ebcli, eb-ssm, and eb-create-environment into requirements.in (surrounded by that feature flag)